projects/plugins/jetpack/extensions/blocks/contact-info/edit.native.js

```diff
@@ -5,13 +5,19 @@ import styles from './editor.scss';
 const ALLOWED_BLOCKS = [
 	'jetpack/address',
 	'jetpack/email',
+	'jetpack/name',
 	'jetpack/phone',
 	'core/heading',
 	'core/separator',
 	'core/spacer',
 ];
 
-const TEMPLATE = [ [ 'jetpack/email' ], [ 'jetpack/phone' ], [ 'jetpack/address' ] ];
+const TEMPLATE = [
+	[ 'jetpack/name' ],
```

Comment:
@enejb, I'm pinging you directly since it seems I can't ping your team.

I've added a `name` field to the native version of the Contact Info block. It seemed straightforward as it was another text field. However, I haven't added the `image` field that's now part of the web version as it looked more involved.

I just wanted to put that under your radar. Let me know if there's something I can do to help.

Reply:
Thanks for the ping. I am going to ask the gutenberg mobile team in slack about this change and for someone to take a closer look. 

I know that the image block is supported in the native editor. So maybe adding support for this block would also be possible. Would it make sense to just allow that block to be used here instead vs a specific Logo block? Do we need finer control over the markup?

Reply:
Hi @monsieur-z , I took a quick look, and we would have to create a native file for the image since the native `MediaPlaceholder` component differs from the web.

> I know that the image block is supported in the native editor. So maybe adding support for this block would also be possible.

I did try using 'core/image` for mobile, but that is not working as-is. I can create a contact block in the native editor, but I'm getting some errors loading a post created in the web editor.

I can continue looking a bit deeper tomorrow.

Reply:
Thanks!

> Do we need finer control over the markup?
I don't think so. We've only added the `itemprop="image"` attribute on the `img` tag. Unless this is relevant for a native app, too, a regular image container would do.

Reply:
I would have asked the same question as well. It feels like it would be easier to maintain this if we relied on core blocks instead of creating new blocks.

Reply:
I agree on going with a core image block here. At the very minimum, it would unblock getting an image into the contact info block. We can evaluate later if a custom image block is needed

Reply:
Since you've put the PR status back to _In Progress_ @jeherve, I realize there might be some confusion. For the web version, I believe a custom block for the logo is a more straightforward implementation (it also matches the current approach with other children blocks). Using a `core/image` block and later parsing the content to add schema markup is tedious and seems brittle to me.

Reply:
Since the other inner blocks follow the same approach, I'd say let's ship this PR as is.